Travis build doesn't pass

It seems we have a "deploy" build that keeps failing on Travis.
The "build" badge on the main page show "failing".

Here is the error:

```
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':cli:publishMavenJavaPublicationToSonatypeRepository'.
> Failed to publish publication 'mavenJava' to repository 'sonatype'
   > Invalid publication 'mavenJava': multiple artifacts with the identical extension and classifier ('jar', 'null').
* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
* Get more help at https://help.gradle.org
Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.3/userguide/command_line_interface.html#sec:command_line_warnings
BUILD FAILED in 4s
12 actionable tasks: 9 executed, 3 up-to-date
The command "./gradlew publishToSonatype -PsonatypeUser=$SONATYPE_USER -PsonatypePassword=$SONATYPE_PASSWORD" exited with 1.
Done. Your build exited with 1.

```

--- a/README.md
+++ b/README.md
@@ -70,9 +70,9 @@ This IDE is cross platform so it should work on any OS. This project was created
 
 `java -jar openapi-style-validator.jar -s ./path/to/spec.yaml -o ./path/to/options.json`
 
-Example using the default output path for the jar:
+Example using the default output path for the jar (replace `<version>` with the current version):
 
-`java -jar modules/cli/build/libs/openapi-style-validator-cli-1.0.jar -s specs/petstore.yaml -o specs/options.json`
+`java -jar modules/cli/build/libs/openapi-style-validator-cli-<version>-all.jar -s specs/petstore.yaml -o specs/options.json`
 
 #### Command Line
 

--- a/cli/build.gradle
+++ b/cli/build.gradle
@@ -37,7 +37,7 @@ shadowJar {
 	//noinspection GroovyAccessibility
     archiveBaseName = "openapi-style-validator-cli"
 	//noinspection GroovyAccessibility
-	archiveClassifier = null
+	archiveClassifier = 'all'
 }
 
 project.tasks.assemble.dependsOn project.tasks.shadowJar

